src/Workload/workloads.csproj

```diff
@@ -0,0 +1,202 @@
+<Project DefaultTargets="Restore;Build">
+  <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.props" />
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+
+    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+    <_RootDir Condition=" '$(BUILD_STAGINGDIRECTORY)' != '' ">$([MSBuild]::EnsureTrailingSlash($(BUILD_STAGINGDIRECTORY)))</_RootDir>
+    <_RootDir Condition=" '$(_RootDir)' == '' ">$(MauiRootDirectory)</_RootDir>
+    <OutputPath>$(_RootDir)\artifacts\</OutputPath>
+    <ManifestOutputPath>$(OutputPath)manifests</ManifestOutputPath>
+    <PreviewManifestOutputPath>$(OutputPath)manifests-pre</PreviewManifestOutputPath>
+    <PackManifestOutputPath>$(OutputPath)manifests-packs</PackManifestOutputPath>
+    <MsiNuGetOutputPath>$(ArtifactsShippingPackagesDir)</MsiNuGetOutputPath>
+    <NuGetPackagePath Condition=" '$(NuGetPackagePath)' == '' ">$(ArtifactsShippingPackagesDir)</NuGetPackagePath>
+    <WorkloadMsiGenProps Condition=" '$(WorkloadMsiGenProps)' == '' ">$(ArtifactsShippingPackagesDir)/vs-workload.props</WorkloadMsiGenProps>
+    <DropBuildNumber Condition=" '$(DropBuildNumber)' == '' ">$(BUILD_BUILDID)</DropBuildNumber>
+    <DropBuildNumber Condition=" '$(BUILD_SOURCEVERSIONSHORT)' != '' ">$(BUILD_SOURCEVERSIONSHORT)/$(DropBuildNumber)</DropBuildNumber>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <WixToolsetPath>$(PkgMicrosoft_Signed_Wix)\tools</WixToolsetPath>
+    <SwixPluginPath>$(PkgMicroBuild_Plugins_SwixBuild_Dotnet)</SwixPluginPath>
+    <SwixBuildTargets>$(SwixPluginPath)\build\MicroBuild.Plugins.SwixBuild.targets</SwixBuildTargets>
+    <SwixTargetsPath>$(PkgMicroBuild_Plugins_SwixBuild_Dotnet)\build\MicroBuild.Plugins.SwixBuild.targets</SwixTargetsPath>
+  </PropertyGroup>
+
+  <Import Project="$(WorkloadMsiGenProps)" />
```

Comment:
If we are no longer going to be sharing this file across repos, we could also get rid of the separate vs-workload.props file which contains product/repo specific information. Those props/items could be moved in here instead.